examples/IMU/Madgwick6DOF/Madgwick6DOF.ino

```diff
@@ -40,8 +39,8 @@ void loop() {
     accelerometer.readAccelerationGXYZ(ax, ay, az);
     // Считываем данные с гироскопа в радианах в секунду
     gyroscope.readRotationRadXYZ(gx, gy, gz);
-    // Устанавливаем коэффициенты фильтра
-    filter.setKoeff(fps, BETA);
+    // Устанавливаем частоту фильтра
+    filter.setFrequency(fps);
```

Comment:
You changed the library interface. Old programs will not be able to work with the new library.

Reply:
Done.